client/pages/Login/Login.jsx

```diff
@@ -0,0 +1,34 @@
+// React lib import
```

Comment:
lets move this to `client/components/Login`. Since this is the root page for the entire login component, you can name it`index.js`. So we should have `client/components/Login/index.js`